### PR TITLE
Change the way that blog page behave, acording to designer feedback

### PR DESCRIPTION
**Pull request by AdamMankowski**
## TO DO:
* [X] Change a way that blogposts behave on the page - width: auto;
* [X] Change font sizes and everything connected to it

---

Review:
* [x] @beatadelura
* [ ] @kruszczynski
* [x] @pjar
* [ ] @rpietraszko
* [ ] @MagdaMalinowska
* [ ] @irene50288

### CODE REVIEW

File: source/stylesheets/modules/_m-grid-item.scss [hunk 1]
```diff
@@ -28,27 +28,27 @@ $color: $breaker-bay-green;
...
-    font-size: 12px;
+    font-size: 13px;
     padding: 7px 0;
   }
 
   .title {
     color: currentColor;
-    font-size: 30px;
+    font-size: 34px;
     line-height: 122%;
```
Comment (beatadelura) [inline source/stylesheets/modules/_m-grid-item.scss]: Please use mixin `font` for all font sizes and line-heights
https://github.com/codequest-eu/website/blob/master/source/stylesheets/base/_mixins.scss#L261

File: source/stylesheets/modules/_m-grid-item.scss [hunk 2]
```diff
@@ -28,27 +28,27 @@ $color: $breaker-bay-green;
...
+    font-size: 17px;
     margin: 12px 0 10px;
   }
 
   .summary {
     color: $martinique-dark;
     font-family: $baskerville;
-    font-size: 15px;
+    font-size: 18px;
     line-height: 180%;
```
Comment (beatadelura) [inline source/stylesheets/modules/_m-grid-item.scss]: same here

File: source/stylesheets/modules/_m-grid-item.scss [hunk 3]
```diff
@@ -57,10 +57,10 @@ $color: $breaker-bay-green;
 
 @media #{$breakpoint-tablet} {
   .m-grid-item {
-    height: 470px;
+    height: 460px;
     margin-top: 54px;
     padding: 0 5px;
-    width: 370px;
+    width: 48%;
```
Comment (beatadelura) [inline source/stylesheets/modules/_m-grid-item.scss]: You can use mixin `size` for width and height https://github.com/codequest-eu/website/blob/master/source/stylesheets/base/_mixins.scss#L242

**Comment by beatadelura** [comment]
Good job :+1:

**Comment by MagdaMalinowska** [comment]
@AdamMankowski did you check https://github.com/codequest-eu/website/pull/143. Let's don't do 2 PR with same purpose in the same time :(

**Comment by AdamMankowski** [comment]
@MagdaMalinowska I did now ;] and it is about something utterly different. I changed a whole layout on main blog page (not so flexy now ;])

**Comment by pjar** [comment]
It would be great to see it working with flex-box, but this will need some work probably

**Pull request merged**